### PR TITLE
Rle 19feb

### PR DESCRIPTION
**Pull request by regilero**

### CODE REVIEW

**Comment by regilero** [comment]
@kiorky, if you want to review, this is on the main D7 version.

File: .salt/280_cron.sls [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
+                LOG="{{cfg.data_root}}/var/log/cron.log"
                 lock="${0}.lock"
-                find "${lock}" -type f -mmin +1 -delete 1>/dev/null 2>&1
+                find "${lock}" -type f -mmin +{{data.cron_lock_minutes}} -delete 1>/dev/null 2>&1
                 if [ -e "${lock}" ];then
                   echo "Locked ${0}";exit 1
                 fi
                 touch "${lock}"
-                {{data.cron_cmd}}
+                {{data.cron_cmd}} 2>&1 > ${LOG}
```
Comment (kiorky) [inline .salt/280_cron.sls]: 2>&1 after stdin redirect

File: .salt/PILLAR.sample [hunk 2]
```diff
@@ -92,7 +96,7 @@ makina-projects.projectname:
       allow_url_fopen: 1
       upload_max_filesize: "500M"
       max_input_vars: 3000
-      fpm_user: "{user}-php"
+      fpm_user: "{name}-php"
```
Comment (kiorky) [inline .salt/PILLAR.sample]: {name} is the name of the project, i think you choosed a bad name for the new "php user", we should clarify that

**Comment by kiorky** [comment]
seems good except the small styles fixes and the user question

File: .salt/PILLAR.sample (hunk 2 above)
Comment (regilero) [inline .salt/PILLAR.sample]: Yes, say your projet is foo, you'll get a php fpm pool for foo with a user `foo-php`, seems better for me.
And you also have `foo-user` as project user (owning files) and `foo-grp` as the group (mainly for shared rights).

If you have several projects, each with one pool, you'll get several `*-php` users.

File: .salt/280_cron.sls (hunk 1 above)
Comment (regilero) [inline .salt/280_cron.sls]: you mean `> ${LOG] 2>&1` ?

**Pull request merged**